### ISSUE TITLE
subprocess options broken with pathos change

### ISSUE THREAD

**Issue by djkapner** [fcollman/render-python#82]
this merge allowed redirection of java output:
https://github.com/fcollman/render-python/pull/71

since the pathos cut commit https://github.com/fcollman/render-python/commit/d6bc7c5f0b99b93ce58319351fda48b087f10c2c
it breaks:
```   
renderapi.client.import_tilespecs_parallel(outputname,tspecs,sharedTransforms=shared_tforms,render=ingestconn,close_stack=False,stderr=stdeo,stdout=stdeo)
  File "<decorator-gen-36>", line 2, in import_tilespecs_parallel
  File "build/bdist.linux-x86_64/egg/renderapi/client.py", line 50, in renderclientaccess
  File "build/bdist.linux-x86_64/egg/renderapi/client.py", line 377, in import_tilespecs_parallel
  File "/home/danielk/anaconda2/envs/emaligner/lib/python2.7/multiprocessing/pool.py", line 253, in map
    return self.map_async(func, iterable, chunksize).get()
  File "/home/danielk/anaconda2/envs/emaligner/lib/python2.7/multiprocessing/pool.py", line 572, in get
    raise self._value
ValueError: I/O operation on closed file
```
this line is now missing, but adding it back in did not fix the problem:
https://github.com/RussTorres/render-python/blob/3774b4c73020a3b5cefcadbad3de10ec50429df3/renderapi/client.py#L487

**Comment by RussTorres** [comment]
The problem here is that filehandlers don't stay open (object repr is `<closed file '<uninitialized file>', mode '<uninitialized file>' at 0x7f0ed2a15390>`) with the new multiprocessing implementation.  I'm not sure what the best way to handle this is -- we can work around it with initializers and globals on a per-process basis or I can push forward on allowing external multiprocessing pool libraries (reintroducing pathos as an option.)

**Comment by fcollman** [comment]
I think this is addressed by #87 @djkapner why don't you try it out, if you are happy you can close the issue.

**Comment by RussTorres** [comment]
Also worth noting: using a `multiprocess.pool.ThreadPool` with an `__enter__` and `__exit__` will keep the filehandlers open as it doesn't need to fork.  I'm not sure how well this works for all our use cases (GIL limited), but it could be useful to avoid the pathos dependency assuming java is doing the heavy lifting.
